cogs/Alerts.py

```diff
@@ -10,7 +10,7 @@
 
 
 def get_keywords(ctx: discord.AutocompleteContext) -> list:
-    conn = sqlite3.connect("util/alerts.db")
+    conn = sqlite3.connect("util/database.sqlite")
```

Comment:
Just kind of curious, why put the database in the `util/` directory? It's not exactly a utility, it's pretty integral to the function of the bot. Maybe makes more sense to just leave it at the top level?

Reply:
That's how it was and I don't dare change it